apps/web/src/components/display/AddRecordModal.tsx

```diff
@@ -1,23 +1,38 @@
 import { useRouter } from 'next/router';
 import { Formik, Form as FormikForm } from 'formik';
 import { Field, Select, Option } from '../form';
+import createValidator from 'class-validator-formik';
 
 import { Modal } from '../Modal';
 import { Button } from '@components';
+import { IENApplicantJobCreateUpdateDTO } from '@ien/common';
+import { addJobRecord, healthAuthority, jobLocations, jobTitle } from '@services';
 
-export const AddRecordModal: React.FC = () => {
+interface AddRecordProps {
+  jobRecords: any;
+  setJobRecords: any;
+}
+
+export const AddRecordModal: React.FC<AddRecordProps> = ({ jobRecords, setJobRecords }) => {
   const router = useRouter();
 
+  const applicantId = router.query.applicantId;
```

Comment:
Just to be safe, it might be good to add optional chaining  here just in case a user ends up on this page without an id.